HISTORY.txt

```diff
@@ -9,45 +9,45 @@ Changes in 4.6.0 (Upcoming)
-* Removed: Modernizr unnecessary due to IE10 being our minimum IE browser requirement.
+* Removed: Modernizr - unnecessary due to IE10 being the minimum IE browser requirement.
 * Changed: The /textpattern/theme/ directory has been renamed to /textpattern/admin-themes/ for stronger differentiation between this and the public-side /themes/ directory planned for Textpattern 4.7. Additionally prevents breakage of preferences panel caused by old, incompatible themes residing in the previously titled admin theme directory during upgrade.
 * Changed: <txp:article_custom /> accepts 'exclude' attribute to exclude articles by a comma-separated id list.
 * Changed: <txp:author /> accepts 'escape' and 'format' attributes.
 * Changed: <txp:category_list /> accepts 'html_id', 'limit' and 'offset' attributes.
-* Changed: <txp:comments_form/>: Deprecated attributes 'isize', 'msgcols', 'msgrows', 'msgstyle', 'previewlabel', 'submitlabel', 'rememberlabel', and 'forgetlabel'.
+* Changed: <txp:comments_form/>: Deprecated 'isize', 'msgcols', 'msgrows', 'msgstyle', 'previewlabel', 'submitlabel', 'rememberlabel', and 'forgetlabel' attributes .
 * Changed: <txp:comments/>, <txp:comments_form/>, <txp:comments_preview/> and <txp:popup_comments/> can be used as container tags.
-* Changed: <txp:comment_email_input /> accepts attribute 'size'.
-* Changed: <txp:comment_message_input /> accepts attributes 'cols', 'rows'. Removed capability to apply an immediate style to the element.
-* Changed: <txp:comment_name_input /> accepts attribute 'size'.
-* Changed: <txp:comment_preview /> accepts attribute 'label'.
-* Changed: <txp:comment_remember /> accepts attributes 'rememberlabel', 'forgetlabel'.
-* Changed: <txp:comment_submit /> accepts attribute 'label'.
-* Changed: <txp:comment_web_input /> accepts attribute 'size'.
+* Changed: <txp:comment_email_input /> accepts 'size' attribute.
+* Changed: <txp:comment_message_input /> accepts 'cols' and 'rows' attributes. Removed capability to apply an immediate style to the element.
+* Changed: <txp:comment_name_input /> accepts 'size' attribute.
+* Changed: <txp:comment_preview /> accepts 'label' attribute.
+* Changed: <txp:comment_remember /> accepts 'rememberlabel' and 'forgetlabel' attributes.
+* Changed: <txp:comment_submit /> accepts 'label' attribute.
+* Changed: <txp:comment_web_input /> accepts 'size' attribute.
 * Changed: <txp:link /> accepts 'id' and 'name' attributes.
 * Changed: <txp:meta_author /> accepts 'escape' and 'format' attributes.
 * Changed: <txp:meta_keywords /> accepts 'escape', 'format' and 'separator' attributes.
-* Changed: <txp:password_protect /> tag can now be used as a container tag.
+* Changed: <txp:password_protect /> can be used as a container tag.
 * Changed: <txp:recent_articles /> accepts 'offset' attribute.
 * Changed: <txp:section_list /> accepts 'html_id', 'limit' and 'offset' attributes.
-* Changed: Deprecated <txp:keywords /> tag. Use <txp:meta_keywords format="" /> instead.
```

Comment:
This line was a shallow attempt at documentation, because the direct analog of the old `<txp:keywords />` tag is `<txp:meta_keywords format="" />`. Without the attribute, you get different output.

Your edit is neater, because such usage should arguably be in the docs. But we have (perhaps wrongly) put documentation in the HISTORY file before: see the Changes in 4.0.6 section where this kind of thing was written:

    Deprecated tag: <txp:s /> replaced by <txp:page_url type="s" />.

I don' t know if anyone reads this HISTORY prior to upgrade, but on the offchance they do, such hints may be worth keeping as a quick reference so upgraders know they may have to make changes to their site tags.

Don't really mind either way. If docs should be confined to the docs, I'm cool with that.